copycat/api/src/main/java/org/apache/kafka/copycat/sink/SinkTaskContext.java

```diff
@@ -59,6 +60,22 @@ public void offset(Map<TopicPartition, Long> offsets) {
     }
 
     /**
+     * Get the topic partition that throws Transient exception during message delivery
+     * @return the topic partition that has error
+     */
+    public TopicPartition erorrTopicPartition() {
+        return errorTopicPart;
```

Comment:
See note I left in the KAFKA-2480 PR. I didn't notice this with the commit that added the `offsets`, but I think we want to hide the accessors